modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+// * observability/network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-commands_{context}"]
+= Network Observability CLI commands
+
+.Syntax
+`netobserv [flows|packets|cleanup] [filters]`
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Option| Description
+
+| `flows`
+| Capture flows information. You can specify an optional interface name as a filter, such as 'netobserv flows br-ex'.
+
+| `packets`
+| Capture packets from a specific protocol or port pair, such as 'netobserv packets tcp,80'.
+
+| `cleanup`
+| Remove netobserv components.
+
+| `version`
+| Print the software version.
+
+| `--help`
```

Comment:
there is `help` 
```sh
./build/oc-netobserv help

Netobserv allows you to capture flow and packets from your cluster.
Find more information at: https://github.com/netobserv/network-observability-cli/

Syntax: netobserv [flows|packets|cleanup] [filters]

options:
  flows      Capture flows information. You can specify an optionnal interface name as filter such as 'netobserv flows br-ex'.
        Options:
          --interfaces: interfaces to monitor
          --enable_pktdrop: enable packet drop (default: true)
          --enable_dns: enable DNS tracking (default: true)
          --enable_rtt: enable RTT tracking (default: true)
          --enable_filter: enable flow filter (default: false)
          --direction: flow filter direction
          --cidr: flow filter CIDR (default: 0.0.0.0/0)
          --protocol: flow filter protocol (default: TCP)
          --sport: flow filter source port
          --dport: flow filter destination port
          --port: flow filter port
          --sport_range: flow filter source port range
          --dport_range: flow filter destination port range
          --port_range: flow filter port range
          --icmp_type: ICMP type
          --icmp_code: ICMP code
          --peer_ip: peer IP
          --action: flow filter action (default: Accept)
  packets    Capture packets from a specific protocol/port pair such as 'netobserv packets tcp,80'.
  cleanup    Remove netobserv components.
  version    Print software version.
```

Reply:
Hmm ok I'm not sure where I got the `--` from, but I will change `help`

Reply:
My doc is n cli has it it needs to go